src/main/java/frc/robot/Constants.java

```diff
@@ -64,31 +64,21 @@ public static final class LimeLightConstants {
     public static final double MID_DISTANCE_SHOOT = 76;
     public static final double TOP_DISTANCE_SHOOT = 42;
   }
-    public static final class Arm {
-    public static final int kSlotIdx = 0;
-    public static final int kPIDLoopIdx = 0;
-    public static final int kTimeoutMs = 30;
-    public static final int ARM_MAX_ANGLE = 270;
-    public static final int ARM_MIN_ANGLE = 0;
-    public static final int ROTATION_TO_DEGREES = 360;
-    public static final int GEAR_RATIO = 16;
-    public static final double ENCODER_RAW_TO_ROTATION = 8132.;
-  }
 
   public static final class IntakeConstants {
```

Comment:
We should ask @kctyy and @char-ly1 (?I think?) to review the changes made in Intake classes.